### DISCUSSION TITLE
is the version text copying to the correct location?

### DISCUSSION THREAD

**Discussion by MrAdam1983** [cryinkfly/Autodesk-Fusion-360-for-Linux#228] [Q&A] [answered]
or is the launcher.sh looking in the wrong location? 

From the script as it downloads.

wget -N -P $HOME/.config/fusion-360/bin https://raw.githubusercontent.com/cryinkfly/Autodesk-Fusion-360-for-Linux/main/files/builds/stable-branch/bin/build-version.txt  
  online_build_version=`cat $HOME/.config/fusion-360/bin/build-version.txt | awk 'NR == 1'
  online_build_insider_version=`cat $HOME/.config/fusion-360/bin/build-version.txt | awk 'NR == 2'
  echo "Online Build-Version: $online_build_version"

then it looks for the version file in a different location with a different file name.....right?

system_build_version=`cat $HOME/.wineprefixes/fusion360/drive_c/users/$USER/AppData/Roaming/Autodesk/Autodesk\ Fusion\ 360/API/version.txt

Either way, the file in the second location does not exist.  I saw a discussion where it was said to create the file but I don't follow the syntax.

I am missing the location 

Autodesk\ Fusion\

which is easy enough to create, but then exactly how is the file supposed to be formatted and created?

360/API/version.txt

Or should that be a full location and a file name such as:

Autodesk\ Fusion360\API\Version.txt

I have run through this several times and it seems to me (as mostly a noob) that it is looking for the same file by 2 different names in 2  different locations
[launcher.txt](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/files/8748022/launcher.txt)
.

**Comment by cryinkfly** [comment, marked as answer]
I have solved this problem with my updated setup wizard. :+1:
